jcl/src/java.base/share/classes/java/lang/Object.java

```diff
@@ -105,23 +103,25 @@ explicitly call runAllFinalizers(). So, he won't get finalization for free,
 	but he can still invoke it. Since this method is empty, the space cost for it
 	is negligible, so we leave it in.
 /*[ENDIF]*/
-/*[IF Sidecar19-SE] */
+/*[IF JAVA_SPEC_VERSION >= 18] */
+@Deprecated(forRemoval=true, since="9")
+/*[ELSEIF JAVA_SPEC_VERSION >= 9] */
```

Comment:
Mismatching comment
```suggestion
/*[ELSEIF JAVA_SPEC_VERSION >= 18] */
```

Reply:
No, that's `ELSEIF` (guarding the version for Java 9-17).